modules/dxp/apps/portal-security-audit/portal-security-audit-web/src/main/java/com/liferay/portal/security/audit/web/internal/display/context/AuditDisplayContext.java

```diff
@@ -353,7 +360,8 @@ private int _getStartDateDay() {
 		}
 
 		_startDateDay = ParamUtil.getInteger(
-			_httpServletRequest, "startDateDay", _today.get(Calendar.DATE));
+			_themeDisplay.getRequest(), "startDateDay",
+			_today.get(Calendar.DATE));
 
```

Comment:
@ChrisKian, the commit will cause new issue. Please see the steps.

1. Go to Audit Portlet,
2. Expand the advanced Search
3. In the Start Date, change date for 08/07/2023
4. Click Search, we can see the result list with pagination.
5. Check the third(3) page url, it has the param **_com_liferay_portal_security_audit_web_portlet_AuditPortlet_startDateDay=7**. And then click 3.

Result: no result returned. That's because `_startDateDay `return the default value `_today.get(Calendar.DATE)`, not 7.

Reply:
Thanks @yuhai, I will try to get to this tomorrow.